runtime/src/main/java/com/dylibso/chicory/runtime/Instance.java

```diff
@@ -217,6 +217,10 @@ public Instance initialize(boolean start) {
             }
         }
 
+        if (start && module.export(START_FUNCTION_NAME) != null) {
```

Comment:
Can you explain this change?

Reply:
This means we are now doing validation before running the start function -- which seems correct.